src/components/AccessibilityRequestsTable/index.tsx

```diff
@@ -262,7 +262,9 @@ const AccessibilityRequestsTable: FunctionComponent<AccessibilityRequestsTablePr
   );
 };
 
-const currentTableSortDescription = headerGroup => {
+const currentTableSortDescription = <T extends {}>(
```

Comment:
Was this to fix a type error?

Reply:
Yeah, without that, there were TS errors due to an implicit `any` type. I had the file open for reference, and it was a pretty easy fix, so I figured I'd throw it in.